metricbeat/module/kibana/kibana.go

```diff
@@ -81,6 +85,12 @@ func IsSettingsAPIAvailable(currentKibanaVersion *common.Version) bool {
 	return elastic.IsFeatureAvailable(currentKibanaVersion, SettingsAPIAvailableVersion)
 }
 
+// IsUsageExcludable returns whether the stats API supports the exclude_usage parameter in the
+// given version of Kibana
+func IsUsageExcludable(currentKibanaVersion *common.Version) bool {
+	return elastic.IsFeatureAvailable(currentKibanaVersion, ExcludeUsageParamAvailableVersion)
```

Comment:
Does it not matter that this parameter won’t exist in 7.0.0?

Reply:
Good point. I could make the check finer grained like `(6.7.3 <= version < 7.0.0) || (version >= 7.0.2)` OR we could just say that Metricbeat supports this PR's feature starting 7.0.2, so `version >= 7.0.2`.

Reply:
Either way work. I was thinking about it though: Kibana just ignores unrecognized parameters, right? So it’s not awful to send the param to `7.0.0` if that’s true (pretty sure it is).

Reply:
> Kibana just ignores unrecognized parameters, right?

Evidently it doesn't. This is basically why this PR's CI is failing right now with a 400 from Kibana because it's seeing an unrecognized parameter. If it were more lenient, we wouldn't have needed any version checks in this code at all — we could've always just sent the parameter.

Reply:
If only we were lazy and lenient! :)

In that case, yeah I’d vote for the right fix that checks for both versions. Since people frequently use newer beats with older stack stuff, I’d vote for the “best” but most complex version (but wait to see if it makes it into 7.0.1 versus 7.0.2?).

Reply:
I implemented the more precise version check but went with `7.0.2` to be conservative.